reference/api/experimental-features.mdx

```diff
@@ -0,0 +1,65 @@
+# Experimental
+
+The `/experimental-features` route configures Meilisearch's [experimental features](/learn/experimental/overview). All operations in this route are synchronous.
```

Comment:
```suggestion
The `/experimental-features` route allows you to activate or deactivate some of Meilisearch's [experimental features](/learn/experimental/overview). All operations in this route are synchronous.
```

This could just be a personal feeling, but I would want to clarify the second sentence. Example:

```suggestion
The `/experimental-features` route allows you to activate or deactivate some of Meilisearch's [experimental features](/learn/experimental/overview).

This route is **synchronous**. This means that no task object will be returned, and any activated or deactivated features will be made available or unavailable immediately.
```